benchmarks/benchmark_base.py

```diff
@@ -62,8 +61,7 @@ def config_rpacket_tracking(self):
     @functools.cached_property
     def tardis_ref_path(self):
         ref_data_path = Path(
-            Path(__file__).parent.parent,
-            "tardis-refdata"
+            Path(__file__).parent.parent, "tardis-refdata"
```

Comment:
That needs to be parameterized and not fixed. The ref-data directory needs to be a config option.

Reply:
I didn't write that function. I just ran black on that file, and that was the change it made.

Reply:
The change I made is from line 204 to 207.

Reply:
okay - talked to @officialasishkumar